### PR TITLE
Markdown render support

### PR DESCRIPTION
**Pull request by Herndl**
Adds markdown to ant design components

### CODE REVIEW

**Comment by csillag** [comment]
What is the ETA for having this merged? I would like to use this feature.

**Comment by scharf** [comment]
...once this is merged, we should create another release...

**Comment by scharf** [comment]
rebased on `master`

File: examples/src/app/ApplicationImpl.ts [hunk 1]
```diff
@@ -49,6 +51,18 @@ export class ApplicationImpl implements Application {
...
+    readonly testLabel: Label = new LabelImpl({
+        id: 'ApplicationImpl.testLabel',
+        text: '<h2>LALALAL</h2>',
+        showRawText: false,
+    });
+
+    readonly testLabelMarkdown: Label = new LabelImpl({
+        id: 'ApplicationImpl.testLabel2',
+        text: '# Hello Button',
+        showRawText: false,
```
[the remark this answers is not in the document]
Comment (Herndl) [reply examples/src/app/ApplicationImpl.ts]: Yes it is the default, this is just a remnant from experimenting and testing 😄

**Pull request merged**